tests/integration/test_scheduler.py

```diff
@@ -86,33 +86,44 @@ async def test_shutdown_CylcError_log(one: Scheduler, run: Callable):
     logged in one line."""
     schd = one
 
+    # patch the shutdown to raise an error
     async def mock_shutdown(*a, **k):
         raise CylcError("Error on shutdown")
     setattr(schd, '_shutdown', mock_shutdown)
 
+    # run the workflow
     log: pytest.LogCaptureFixture
     with pytest.raises(CylcError) as exc:
         async with run(schd) as log:
             pass
+
+    # check the log records after attempted shutdown
     assert str(exc.value) == "Error on shutdown"
     last_record = log.records[-1]
     assert last_record.message == "CylcError: Error on shutdown"
     assert last_record.levelno == logging.ERROR
 
+    # shut down the scheduler properly
+    await Scheduler._shutdown(schd, Exception('because I said so'))
```

Comment:
This is cleaner in the log, less likely to distract if debugging failure due to some other reason (may have to import `SchedulerStop`)
```suggestion
    await Scheduler._shutdown(schd, SchedulerStop('because I said so'))
```